tests/pds/dataservice_test.go

```diff
@@ -1157,8 +1157,21 @@ var _ = Describe("{DeployAllDataServices}", func() {
 	})
 })
 
-func TriggerDeployDataService(ds PDSDataService, namespace, tenantID, projectID string) (*pds.ModelsDeployment, map[string][]string, map[string][]string, error) {
+func TriggerDeployDataService(ds PDSDataService, namespace, tenantID, projectID string, deployOldVersion bool) (*pds.ModelsDeployment, map[string][]string, map[string][]string, error) {
```

Comment:
Function UpgradeDataService is doing a similar task. Let's try to reuse either that or make changes in that to re-use this code.

Reply:
yes I have checked it, but UpgradeDataService function is having lot of validations which is not required and the easiest way of implementation is with this method.

Reply:
Ok, in that case let's reuse this code in UpgradeDataService function.